client/src/app/Constants.ts

```diff
@@ -14,13 +14,16 @@ import {
   ProposedAction,
   Risk,
 } from "@app/api/models";
-
 import { ENV } from "./env";
 
+export enum BrandType {
+  Konveyor = "konveyor",
+  MTA = "mta",
+}
+
 export const isAuthRequired = ENV.AUTH_REQUIRED !== "false";
 export const uploadLimit = ENV.UI_INGRESS_PROXY_BODY_SIZE || "500m";
-export const isPropietaryAllowed =
-  ENV.PROFILE === "konveyor" || ENV.PROFILE === "";
```

Comment:
By default the app is in upstream context with PROFILE being either "konveyor" or not defined ("").
By not allowing not defined, the logic is changing a bit here.
Maybe that's because initially the values for PROFILE envar weren't fully known.
But from an open source project viewpoint I think it's important we keep that way, so the brand could be anything.